api/cloudcontroller/ccv3/service_instance.go

```diff
@@ -122,22 +123,58 @@ func (client *Client) UnshareServiceInstanceFromSpace(serviceInstanceGUID string
 
 // GetServiceInstanceSharedSpaces will fetch relationships between
 // a service instance and the shared-to spaces for that service.
-func (client *Client) GetServiceInstanceSharedSpaces(serviceInstanceGUID string) ([]resources.Space, Warnings, error) {
-	var relationships resources.RelationshipList
-
+func (client *Client) GetServiceInstanceSharedSpaces(serviceInstanceGUID string) ([]SpaceWithOrganization, Warnings, error) {
+	var responseBody resources.SharedToSpacesListWrapper
+	query := []Query{
+		{
+			Key:    FieldsSpace,
+			Values: []string{"guid", "name", "relationships.organization"},
+		},
+		{
+			Key:    FieldsSpaceOrganization,
+			Values: []string{"guid", "name"},
+		},
+	}
 	_, warnings, err := client.MakeRequest(RequestParams{
 		RequestName:  internal.GetServiceInstanceRelationshipsSharedSpacesRequest,
 		URIParams:    internal.Params{"service_instance_guid": serviceInstanceGUID},
-		ResponseBody: &relationships,
+		Query:        query,
+		ResponseBody: &responseBody,
+	})
+	return mapRelationshipsToSpaces(responseBody), warnings, err
+}
+
+func (client *Client) GetServiceInstanceUsageSummary(serviceInstanceGUID string) ([]resources.ServiceInstanceUsageSummary, Warnings, error) {
+	var result resources.ServiceInstanceUsageSummaryList
+
+	_, warnings, err := client.MakeRequest(RequestParams{
+		RequestName:  internal.GetServiceInstanceSharedSpacesUsageSummaryRequest,
+		URIParams:    internal.Params{"service_instance_guid": serviceInstanceGUID},
+		ResponseBody: &result,
 	})
+	return result.UsageSummary, warnings, err
+}
 
-	return mapRelationshipsToSpaces(relationships), warnings, err
+type SpaceWithOrganization struct {
```

Comment:
For types like this one that are exposed outside the actor, I have a slight preference for them to be defined at the top of the file.